tests/utils/test_endpoints.py

```diff
@@ -144,3 +145,44 @@ def test_read_endpoint_config(filename: Text, endpoint_type: Text):
 def test_read_endpoint_config_not_found(filename: Text, endpoint_type: Text):
     conf = endpoint_utils.read_endpoint_config(filename, endpoint_type)
     assert conf is None
+
+
+@pytest.mark.parametrize(
+    "value, default, expected_result",
+    [
+        (None, True, True),
+        (False, True, False),
+        ("false", True, False),
+        ("true", False, True),
+    ],
+)
+def test_bool_arg(value, default, expected_result):
```

Comment:
can you please add type annotations to the test parameters? (same for the other test functions) 🙌🏻

Reply:
Sure